src/download.rs

```diff
@@ -4,11 +4,13 @@ use std::io;
 use std::fs::File;
 use std::path::Path;
 use log::info;
-use url::Url;
-
 use reqwest::StatusCode;
-
 use sha2::{Sha256, Digest};
+use tokio::time::Duration;
+use url::Url;
+
+const MAX_DOWNLOAD_RETRY: u32 = 10;
+const RETRY_INTERVAL_MSEC: u64 = 500;
```

Comment:
```suggestion
const MAX_DOWNLOAD_RETRY: u32 = 20;
const RETRY_INTERVAL_MSEC: u64 = 1000;
```
How about increasing this a bit, otherwise it's done retrying after 5 secs